puppet/modules/quickstack/data/RedHat.yaml

```diff
@@ -57,6 +59,14 @@ quickstack::pacemaker::cinder::backend_rbd_name: rbd_backend
 quickstack::pacemaker::cinder::db_ssl: 'false'
 quickstack::pacemaker::cinder::db_ssl_ca: "%{hiera('db_ca')}"
 quickstack::pacemaker::cinder::debug: 'false'
+quickstack::pacemaker::cinder::dell_sc_san_ip: ['']
+quickstack::pacemaker::cinder::dell_sc_san_login: ['Admin']
+quickstack::pacemaker::cinder::dell_sc_iscsi_ip_address: ['192.168.0.20']
+quickstack::pacemaker::cinder::dell_sc_iscsi_port: ['3260']
+quickstack::pacemaker::cinder::dell_sc_ssn: ['64702']
+quickstack::pacemaker::cinder::dell_sc_api_port: ['3033']
+quickstack::pacemaker::cinder::dell_sc_server_folder: ['server']
+quickstack::pacemaker::cinder::dell_sc_volume_folder: ['volume']
```

Comment:
Since we are placing Dell_SC after EQL in other files and docs can we keep consistent in astapor also?